### PR TITLE
Es6 template literals

### PR DESCRIPTION
**Pull request by OSAdams**
image

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log(`LearningFuze: ${address}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Please don't modify anything which isn't concatenation:

---

image

---

image

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit} ${item}`;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Please don't add or remove any characters when you convert concatenation to template literals:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by OSAdams** [comment]
image

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    console.log('Bio: ', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Please don't change anything which isn't concatenation:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by OSAdams** [comment]
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**